infra/modules/azure_api_management/variables.tf

```diff
@@ -0,0 +1,386 @@
+      body_bytes     = number      # body_bytes - (optional) is a type of number
+      headers_to_log = set(string) # headers_to_log - (optional) is a type of set of string
+    }
+  )
+  default = null
+}
+
+variable "diagnostic_frontend_response" {
+  description = "Number of payload bytes to log (up to 8192) and a list of headers to log"
+  type = object(
+    {
+      body_bytes     = number      # body_bytes - (optional) is a type of number
+      headers_to_log = set(string) # headers_to_log - (optional) is a type of set of string
+    }
+  )
+  default = null
+}
+
+
+variable "metric_alerts" {
+  default = {}
+
+  description = <<EOD
+Map of name = criteria objects
+EOD
+
+  type = map(object({
+    description = string
+    # Possible values are PT1M, PT5M, PT15M, PT30M and PT1H
+    frequency = string
+    # Possible values are PT1M, PT5M, PT15M, PT30M, PT1H, PT6H, PT12H and P1D.
+    window_size = string
+    # Possible values are 0, 1, 2, 3.
+    severity = number
+    # Possible values are true, false
+    auto_mitigate = bool
+
+    criteria = set(object(
+      {
+        # criteria.*.aggregation to be one of [Average Count Minimum Maximum Total]
+        aggregation = string
+        dimension = list(object(
+          {
+            name     = string
+            operator = string
+            values   = list(string)
+          }
+        ))
+        metric_name      = string
+        metric_namespace = string
+        # criteria.0.operator to be one of [Equals NotEquals GreaterThan GreaterThanOrEqual LessThan LessThanOrEqual]
+        operator               = string
+        skip_metric_validation = bool
+        threshold              = number
+      }
+    ))
+
+    dynamic_criteria = set(object(
+      {
+        # criteria.*.aggregation to be one of [Average Count Minimum Maximum Total]
+        aggregation       = string
+        alert_sensitivity = string
+        dimension = list(object(
+          {
+            name     = string
+            operator = string
+            values   = list(string)
+          }
+        ))
+        evaluation_failure_count = number
+        evaluation_total_count   = number
+        ignore_data_before       = string
+        metric_name              = string
+        metric_namespace         = string
+        operator                 = string
+        skip_metric_validation   = bool
+      }
+    ))
+  }))
+}
+
+variable "action" {
+  description = "The ID of the Action Group of custom string properties to include with the post webhook operation."
+  type = set(object(
+    {
+      action_group_id = string
+    }
+  ))
+  default = []
+}
+
+variable "alerts_enabled" {
+  type        = bool
+  default     = true
+  description = "Should Metrics Alert be enabled?"
+}
+
+variable "sec_log_analytics_workspace_id" {
+  type        = string
+  default     = null
+  description = "Log analytics workspace security (it should be in a different subscription)."
+}
+
+variable "sec_storage_id" {
```

Comment:
What is it?

Reply:
is used in the resource `azurerm_monitor_diagnostic_setting`, from [here](https://github.com/pagopa/terraform-azurerm-v3/blob/ab2cf6a43414f2cc80a9e51332182c26ad970f72/api_management/main.tf#L355)

Reply:
What we generally use as value for this one? I can't find its value in `io-infra`?

Reply:
It was never called and the default is null, so i've removed that part